src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/ArtworkFilterArtworkGrid.tsx

```diff
@@ -1,8 +1,8 @@
-import { Box, Message, Spacer } from "@artsy/palette"
+import { Box, Spacer } from "@artsy/palette"
 import { ArtworkFilterArtworkGrid_filtered_artworks } from "__generated__/ArtworkFilterArtworkGrid_filtered_artworks.graphql"
 import { FilterState } from "Apps/Artist/Routes/Overview/state"
 import { ContextConsumer } from "Artsy/Router"
-import ArtworkGrid from "Components/ArtworkGrid"
+import ArtworkGrid from "Components/ArtworkGrid/ArtworkGrid"
```

Comment:
As I mentioned offline, I think we could structure this module differently. By using `ArtworkGrid/index.ts` similar to (https://github.com/artsy/reaction/blob/master/src/Apps/Artist/Routes/Articles/index.tsx)
since we don't want to use default exports anymore.

Anyway this would require some refactoring and we're on a tight deadline we.

Reply:
The default export is actually because it is being wrapped in a styled component, can change to named export.

Reply:
@eessex - Just a heads up that removing this file and moving into a folder broke a few things in Force that imported ArtworkGrid directly from the `/Components` folder. For refactors like this a better way to go about it is to create a new folder of the same name as the file being removed and then add an `index` file inside -- so it'd be like `Components/ArtworkGrid/index.tsx` as the structure. That way there's backward compat for other consumers.